spec/controllers/ems_common_controller_spec.rb

```diff
@@ -43,6 +43,25 @@ def set_up_controller_for_show_method(controller, ems_id)
       end
     end
 
+    context "#new" do
+      before do
+        set_user_privileges
+        allow(controller).to receive(:drop_breadcrumb)
+      end
+
+      it "assigns provider_regions" do
+        controller.send(:new)
+
+        regions = {
+          # FIXME: (durandom) add a mock provider in order to remove this dependency on an actual provider
+          'azure' => ManageIQ::Providers::Azure::Regions.all.sort_by { |r| r[:description] }.map do |r|
+            [r[:description], r[:name]]
+          end
```

Comment:
It's unfortunate the test code duplicates the UI code.  Not sure how to do it otherwise though.